packages/webdriverjs/src/index.ts

```diff
@@ -174,7 +176,15 @@ export default class AxeBuilder {
       return this.runLegacy(context);
     }
 
-    const partials = await this.runPartialRecursive(context, true);
+    // ensure we fail quickly if an iframe cannot be loaded (instead of waiting
+    // the default length of 30 seconds)
+    const { pageLoad } = await this.driver.manage().getTimeouts();
+    this.driver.manage().setTimeouts({ pageLoad: 1000 });
+    this.frameStack = [];
+
+    const partials = await this.runPartialRecursive(context, null, true);
+
+    this.driver.manage().setTimeouts({ pageLoad });
```

Comment:
Consider wrapping L185 in a try/finally and put restoring the user's driver state in a finally block? Would be very confusing for a user if an error in one axe scan caused subtle timeout behavior changes in the user's subsequent tests